frontend/src/components/SideBar.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import {ReactComponent as HomeIcon} from '../assets/icons/home.svg';
+import {ReactComponent as SetlistIcon} from '../assets/icons/list.svg';
+import TrackIcon from './icons/TracksIcon'
+import ArtistIcon from './icons/ArtistIcon'
+
+const IconClassName = 'icon';
+
+export default class Sidebar extends Component {
```

Comment:
Commented this elsewhere but it got lost:
use this syntax for components:
`const Sidebar: React.FC = () => {}`
`export default Sidebar`

Reply:
The syntax for pages should be like `export default function Home()`

Reply:
Just changes Sidebar.tsx so it follows this Syntax. could you check it's okay and i can update all the others?

Reply:
Just updated SideBar.tsx to follow this Syntax, could you check it's correct the i can do the rest.